scout/server/blueprints/public/templates/public/index.html

```diff
@@ -20,12 +20,12 @@ <h1 class="display-4">Scout</h1>
         </a>
       </p>
     {% else %}
-      {% if config.GOOGLE %}
-        <a class="btn btn-primary btn-lg text-white" href="{{ url_for('login.login') }}" role="button">
-          Login with Google
-        </a>
-      {% elif config.LDAP_HOST or config.LDAP_SERVER %}
-        <form class="row" method="POST" action="{{ url_for('login.login') }}">
+      <form class="row" method="POST" action="{{ url_for('login.login') }}">
```

Comment:
This was a get request before, but I need everything to be in a form in order to submit also the `consent_checkbox`  with the form. So it becomes a POST request in all cases: LDAP/Google and simple login